features/networking/ipv6_dualstack.feature

```diff
@@ -80,3 +79,35 @@ Feature: ipv6 dual stack cluster test scenarios
     Then the step should fail
 
 
+  # @author zzhao@redhat.com
+  # @case_id OCP-46816
+  @admin
+  @network-ovnkubernetes
+  @4.10 @4.9
+  @baremetal-ipi
+  Scenario: ipv6 for nodeport service
```

Comment:
Look like you only cover ipv6 testing here, for dual stack cluster, suggest to cover both ipv4 and ipv6 testing at same time in this script. Otherwise you need another script to cover dual stack testing

Reply:
hmm, it's also ok.  since there is another nodeport is for ipv4.  I just thought it can be covered that.  but sure, we can check both again in this case
updated.